crates/common/tedge_config/src/tedge_toml/tedge_config_location.rs

```diff
@@ -346,6 +344,14 @@ fn update_with_environment_variables(
     Ok(())
 }
 
+fn parse_key_without_warnings(tedge_key: &str) -> Result<WritableKey, ParseKeyError> {
+    tracing::subscriber::with_default(NoSubscriber::new(), || {
+        tedge_key
+            .to_ascii_lowercase()
+            .parse::<tedge_config::WritableKey>()
+    })
+}
```

Comment:
I didn't fully understand this fix, as it seems to ignore all warnings while reading the provided key: not just unknown ones, but even the ones that we have actually deprecated in the past. What am I missing here? Aren't there some old deprecated keys for which we want the deprecation warning to be emitted?